templates/components/molecules/media-block-row/media-block-row.twig

```diff
@@ -6,7 +6,7 @@
 
   {% if items not empty %}
     {% for item in items %}
-      {% include "@decanter/components/molecules/media-block/media-block.twig" with item %}
+      {% include "../../molecules/media-block/media-block.twig" with item %}
```

Comment:
Where did the namespace go? This is important to keep if we plan to use the templates in other systems.